Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +978,54 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandBootstrapOption {
```

Comment:
If we leave this as a global type, we'd need to prefix this with `NIO` because of our [public API declaration](https://github.com/apple/swift-nio/blob/master/docs/public-api.md).

Arguably, we should probably just nest this into `ServerBootstrap` by putting a

```
extension ServerBootstrap {
   [...]
}
```

around this type.

Reply:
Nested as suggested.